scripts/homepage.js

```diff
@@ -227,19 +228,22 @@ $(function () {
     // Change Button text on payment click
     function updateDownloadButton () {
         var payment_amount = $('#amount-custom').val() * 100;
-        if (
-            $('input#amount-custom').hasClass('checked') &&
-            payment_amount < payment_minimum &&
-            $('#amount-custom').val() != ''
-        ) {
+        if ($('#amounts').children().length <= 1) {
             $('#download').text('Download elementary OS');
         } else {
-            $('#download').text('Purchase elementary OS');
+            if (($('input.button#amount-custom').hasClass('checked') &&
+                 $('input.button#amount-custom').val() >= 1) || (
+                 $('button.payment-button').hasClass('checked'))) {
```

Comment:
```
            if (($('input.button#amount-custom').hasClass('checked') &&
                 $('input.button#amount-custom').val() >= 1) || (
                 $('button.payment-button').hasClass('checked'))) {
```

should be formatted as

```
            if (
                (
                    $('input.button#amount-custom').hasClass('checked') &&
                    $('input.button#amount-custom').val() >= 1
                ) || (
                    $('button.payment-button').hasClass('checked')
                )
            ) {
```

Brackets in `if`s should be easy to follow and group from the indentation.